neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -713,13 +722,8 @@ def _launch_process(self):
         stderr_fp = open(self.stderr, "w")
         s3_gw_env = {
             "S3_GW_LISTEN_DOMAINS": self.neofs_env.domain,
```

Comment:
Missed it, could we completely remove ENV vars for the gate? I mean move this setting also to yaml config file

Reply:
fixed